### PR TITLE
Twitter Scraper using snscrape

### LINKED ISSUE #869: Twitter hastag scrapper without API
**Describe the solution you'd like**
This script allows user to enter hastag and scrape the no. of tweets specified by the user and save it in a csv file by using SNScrape library of python.
I am a GSSOC21 participant kindly assign me the same. @kaustubhgupta @avinashkranjan

### PR DESCRIPTION
**Pull request by RohiniRG**
# Description

Using Python's snscrape, we scrape tweets associated with a particular hashtag. Snscrape is a python library that scrapes twitter without the intervention of tweepy.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
- [x] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Twitter_Scraper_without_API/display_hashtags.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import sqlite3
+import os
+
+
+def sql_connection():
+    """
+    Establishes a connection to the SQL file database
+    :return connection object:
+    """
+    path = os.path.abspath('TwitterDatabase.db')
```
Comment (kaustubhgupta) [inline Twitter_Scraper_without_API/display_hashtags.py:10]:
```suggestion
    path = os.path.abspath('./Twitter_Scraper_without_API/TwitterDatabase.db')
```

File: Twitter_Scraper_without_API/fetch_hashtags.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import snscrape.modules.twitter as sntweets
+import sqlite3
+
+
+def sql_connection():
+    """
+    Establishes a connection to the SQL file database
+    :return connection object:
+    """
+    con = sqlite3.connect('TwitterDatabase.db')
```
Comment (kaustubhgupta) [inline Twitter_Scraper_without_API/fetch_hashtags.py:10]:
```suggestion
    con = sqlite3.connect('./Twitter_Scraper_without_API/TwitterDatabase.db')
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Perfect, just a minor suggestion:

**Comment by RohiniRG** [comment]
@kaustubhgupta made the requested changes, have a look

**Review by kaustubhgupta — APPROVED** [review]
💯

**Review by Kushal997-das — APPROVED** [review]
LGTM 👍

**Review by santushtisharma10 — APPROVED** [review]
Nice @RohiniRG 💯

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**